honeybee_schema/radiance/material.py

```diff
@@ -0,0 +1,280 @@
+"""Material Schema"""
+from pydantic import Field, constr
+from typing import List, Union
+
+from .._base import IDdBaseModel
+
+
+class Void(IDdBaseModel):
+    """Void modifier"""
+
+    type: constr(regex='^Void$') = 'Void'
+
+
+class ModifierBase(IDdBaseModel):
+    """Base class for Radiance Modifiers"""
+
+    # TODO: We can't define union of schema objects that haven't been
+    # defined yet. We also can't move this down, because ModifierBase
+    # is inherited by all these objects.
+    modifier: Union[
+         Void, Plastic, Glass, BSDF, Glow, Light, Trans
+    ] = Field(
+        # TODO: This needs to be fixed. How can we add add default=Void,
+        # when Void is not serializable?
+        default='void',
+        description='Material modifier (default: Void).'
+        )
+
+    # TODO: We can't define union of schema objects that haven't been
+    # defined yet. We also can't move this down, because ModifierBase
+    # is inherited by all these objects.
+    dependencies: List[
+        Union[Void, Plastic, Glass, BSDF, Glow, Light, Trans]
+    ] = Field(
+        default=[],
+        min_items=1,
```

Comment:
Shouldn't `min_items` be 0?  Most modifier objects won't have any dependencies.

Reply:
Right, so I guess min_items doesn't need to be defined at all.